testdata/ruby/mistakes/importMissing/test/import_missing_test.rb

```diff
@@ -2,38 +2,26 @@
 require_relative '../lib/import_missing'
```

Comment:
This is missing the `test_init.rb` import

`require_relative 'test_init.rb'`

Reply:
I checked the master branch and this is also true for all the other mistakes and transpile tests for ruby.

They all import `require 'minitest/autorun'` instead of `test_init.rb`. Could you please fix all of those too